spyder/utils/help/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Help Utilities License Information
+##################################
+
+Source files in this directory originate from sources other than Spyder itself,
```

Comment:
This only applies to our MathJax copy, or am I mistaken?

Reply:
Sorry, some files were also taken from somewhere else.

Reply:
Done as above.